src/main/java/com/stripe/model/Customer.java

```diff
@@ -215,4 +215,12 @@ public Customer update(Map<String, Object> params, RequestOptions options)
         Customer.class, options);
   }
   // </editor-fold>
+
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class NextRecurringCharge extends StripeObject {
```

Comment:
I feel like this is the kind of property we should stop supporting fully. I did not even know this existed